pages/docs/react/latest/installation.mdx

```diff
@@ -8,8 +8,8 @@ canonical: "/docs/react/latest/installation"
 
 **Requirements:**
 
-- `rescript@9.1.2` or later (alternatively for older codebases: `bs-platform@8.3 or later`)
-- `react@16.8.1` or later
+- `rescript@10.1` or later
+- `react@18.0.0` or later
 
 Add following dependency to your ReScript project (in case you don't have any project yet, check out the [installation instructions](/docs/manual/latest/installation) in the manual):
```

Comment:
```suggestion
Add the following dependency to your ReScript project (in case you don't have any project yet, check out the [installation instructions](/docs/manual/latest/installation) in the manual):
```